src/components/SearchInput.js

```diff
@@ -1,119 +0,0 @@
-/*
```

Comment:
I think this file is still being used by `SearchableSelect`, which is eventually consumed by the CrisisTemplateContainer. It should eventually be replaced by LUK, but for now I believe it's necessary.

Reply:
I'm pretty sure it's not.